tests/bcl-test/templates/tvOSApp/ViewController.cs

```diff
@@ -85,7 +85,7 @@ public async override void ViewDidLoad ()
 				runner.SkipTests (skippedTests);
 			}
 
-			runner.Run (testAssemblies);
+			await runner.Run (testAssemblies);
```

Comment:
You moved the `TerminateWithSuccess ()` into `BeginInvokeOnMainThread ()`, so I suppose we don't need to continue on the main thread here?

```suggestion
			await runner.Run (testAssemblies).ConfigureAwait (false);
```

Reply:
You are 100% right, good catch (I was working late).